maestro-client/src/main/java/maestro/drivers/AndroidDriver.kt

```diff
@@ -271,26 +271,26 @@ class AndroidDriver(
         val deviceInfo = deviceInfo()
         when (swipeDirection) {
             SwipeDirection.UP -> {
-                val startX = deviceInfo.widthPixels / 2
-                val startY = deviceInfo.heightPixels
-                val endY = deviceInfo.heightPixels / 2
+                val startX = deviceInfo.widthGrid / 2
+                val startY = deviceInfo.heightGrid / 1
+                val endY = deviceInfo.heightGrid / 2
                 dadb.shell("input swipe $startX $startY $startX $endY $durationMs")
             }
             SwipeDirection.DOWN -> {
-                val startX = deviceInfo.widthPixels / 2
-                val startY = 0
-                val endY = deviceInfo.heightPixels / 2
+                val startX = deviceInfo.widthGrid / 2
+                val startY = deviceInfo.widthGrid / 2
```

Comment:
Shouldn't this be based on the `heightGrid`?

Reply:
Yes 🤦 fixing.